multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/util/ApplicationConfiguration.java

```diff
@@ -747,6 +758,12 @@ private Boolean isBasicAuthEnabledThroughEnvironment() {
         return value;
     }
 
+    private Boolean isGitDeploymentEnabledThroughEnvironment() {
```

Comment:
Why does the method name have "git deployment" when it's checking for archive signature verification?